package.json

```diff
@@ -2,7 +2,7 @@
   "name": "dev-portal",
   "private": "true",
   "scripts": {
-    "build:deploy-preview": "hc-tools ./scripts/content-repo-preview/build.ts",
+    "build:deploy-preview": "npx --package @hashicorp/platform-tools hc-tools ./scripts/content-repo-preview/build.ts",
```

Comment:
Since this is running before `npm install`, does it make sense to pin the version of `@hashicorp/platform-tools` so that we can safely release breaking changes of the package without also breaking the build?

Reply:
Ah good call, this is probably the right posture to take in this scenario 👍